CMakeLists.txt

```diff
@@ -155,7 +159,7 @@ target_include_directories(jakt_stage1
 )
 add_executable(Jakt::jakt_stage1 ALIAS jakt_stage1)
 apply_output_rules(jakt_stage1)
-if (MSVC)
+if (MSVC OR (DEFINED MSVC_VERSION AND CMAKE_CXX_COMPILER_ID STREQUAL "Clang"))
```

Comment:
you're looking for `CMAKE_CXX_COMPILER_FRONTEND_VARIANT`

https://gitlab.kitware.com/cmake/cmake/-/issues/22666

https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_FRONTEND_VARIANT.html